### PR TITLE
[stable-2.17] package_facts: ignore warnings by apk on stderr

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Ignore warnings sent by apk cli on stderr

Partial backport of https://github.com/ansible/ansible/pull/83149

Fixes: #83501

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

File: lib/ansible/modules/package_facts.py [hunk 1]
```diff
@@ -440,7 +440,7 @@ class APK(CLIMgr):
 
     def list_installed(self):
         rc, out, err = module.run_command([self._cli, 'info', '-v'])
-        if rc != 0 or err:
+        if rc != 0:
```
Comment (mattclay) [inline lib/ansible/modules/package_facts.py:443]: This change isn't in devel. Why wasn't this change made there first?

Also, what's the relationship between this change and the integration test changes?

Comment (Akasurde) [reply lib/ansible/modules/package_facts.py:443]: The change is available in `devel` via https://github.com/ansible/ansible/pull/83132 

The integration test makes sure that package fact dict is populated `ansible_facts.packages | length > 1`

Comment (mattclay) [reply lib/ansible/modules/package_facts.py:443]: Ah, I had missed that since the module change and the tests were from separate PRs. Thanks.

**Review by mattclay — APPROVED** [review]

File: lib/ansible/modules/package_facts.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/package_facts.py:443]: No problem. Thanks.

**Pull request merged**